numba/cuda/extending.py

```diff
@@ -0,0 +1,7 @@
+"""
+Added for symmetry with the core API
```

Comment:
Should the rest of `numba.extending.overload*` also perhaps be added?

Reply:
I'd like to add it, but I'd like to do it incrementally with tests for each component that I add here in future PRs, rather than adding it all at once.

Reply:
OK, sounds like a safe plan!